src/Canvas.js

```diff
@@ -284,25 +284,29 @@ const Canvas = React.createClass({
     );
   },
 
-  render(): ?ReactElement {
+  render() {
     let displayStart = this.state.displayStart;
     let displayEnd = this.state.displayEnd;
     let rowHeight = this.props.rowHeight;
     let length = this.props.rowsCount;
 
     let rows = this.getRows(displayStart, displayEnd)
-        .map((r, idx) => this.renderRow({
-          key: displayStart + idx,
-          ref: idx,
-          idx: displayStart + idx,
-          row: r.row,
-          height: rowHeight,
-          columns: this.props.columns,
-          isSelected: this.isRowSelected(r.row),
-          expandedRows: this.props.expandedRows,
-          cellMetaData: this.props.cellMetaData,
-          subRowDetails: r.subRowDetails
-        }));
+        .map((r, idx) => {
+          if (r.row !== undefined) {
```

Comment:
@malonecj the filter row height must be adjusted just because of the dropdowns for AutoComplete filter.